tests/User/AutoLoginMiddlewareTest.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+
+namespace Yiisoft\Yii\Web\Tests\User;
+
+use Nyholm\Psr7\Response;
+use PHPUnit\Framework\TestCase;
+use PHPUnit_Framework_MockObject_MockObject;
+use Psr\Http\Message\ResponseInterface;
```

Comment:
not used

Reply:
Some were not used, others if and if I removed them, the tests failed.